cxx-squid/src/test/resources/parser/own/C++11/list-initialization.cc

```diff
@@ -13,7 +13,7 @@ Object anArray[] = {{13.4f, 3}, {43.28f, 29}, {5.934f, 17}}; // An array of thre
 
 class SequenceClass {
 public:
-    SequenceClass(std::initializer_list<int> list);
+    SequenceClass(std::initializer_list<int> list){};
```

Comment:
Yep, my change is working.
It's just to fix compilation of this file by real compiler.